manual/git-disable-gcm/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,5 @@
+﻿$ErrorActionPreference = 'Stop'
+
+Update-SessionEnvironment
+
+git config --system --unset credential.helper
```

Comment:
should probably parse the registry or something for the git executable, just in case the user doesn't it available in PATH.

also, as per [2.2 Formatting](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting) you're missing an empty line at the end of the file